src/Compilers/Core/Portable/Symbols/INamedTypeSymbol.cs

```diff
@@ -186,5 +186,18 @@ public interface INamedTypeSymbol : ITypeSymbol
         /// True if the type is serializable (has Serializable metadata flag).
         /// </summary>
         bool IsSerializable { get; }
+
+        /// <summary>
+        /// True if the type represents a native integer. In C#, the types represented
+        /// by language keywords 'nint' and 'nuint'.
+        /// </summary>
+        bool IsNativeInteger { get; }
```

Comment:
>IsNativeInteger [](start = 13, length = 15)

It looks like IsTupleType is member of ITypeSymbol, should this API be moved there as well?